src/dbos-runtime/debug.ts

```diff
@@ -1,20 +1,36 @@
 import { DBOSConfig, DBOSExecutor } from "../dbos-executor";
+import { GlobalLogger } from "../telemetry/logs";
 import { DBOSRuntime, DBOSRuntimeConfig,  } from "./runtime";
 
 export async function debugWorkflow(dbosConfig: DBOSConfig, runtimeConfig: DBOSRuntimeConfig, proxy: string, workflowUUID: string) {
-  const provDB = `${dbosConfig.poolConfig.database}_prov`;
-  dbosConfig = {...dbosConfig, debugProxy: proxy, system_database: provDB };
-  dbosConfig.poolConfig.database = provDB;
+  dbosConfig = {...dbosConfig, debugProxy: proxy};
+  const logger = new GlobalLogger();
+  try {
+    // Load classes
+    const classes = await DBOSRuntime.loadClasses(runtimeConfig.entrypoint);
+    const dbosExec = new DBOSExecutor(dbosConfig);
+    await dbosExec.init(...classes);
 
-  // Load classes
-  const classes = await DBOSRuntime.loadClasses(runtimeConfig.entrypoint);
-  const dbosExec = new DBOSExecutor(dbosConfig);
-  await dbosExec.init(...classes);
+    // Invoke the workflow in debug mode.
+    const handle = await dbosExec.executeWorkflowUUID(workflowUUID);
+    await handle.getResult();
 
-  // Invoke the workflow in debug mode.
-  const handle = await dbosExec.executeWorkflowUUID(workflowUUID);
-  await handle.getResult();
-
-  // Destroy testing runtime.
-  await dbosExec.destroy();
+    // Destroy testing runtime.
+    await dbosExec.destroy();
+  } catch (e) {
+    const errorLabel = `Debug mode failed`;
+    logger.error(`${errorLabel}: ${(e as Error).message}`);
+    if (e instanceof AggregateError) {
+      console.error(e.errors);
```

Comment:
can we use `logger.error` here?

Reply:
We don't want the stack trace to be here so I had to use `console.error`. In the future we can have more customized solution.